### PR TITLE
Update bin/get_workspace_status given the introduction of pkg/version

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: bin/get_workspace_status [hunk 1]
```diff
@@ -51,14 +51,10 @@ if [[ -n ${ISTIO_DOCKER_HUB} ]]; then
   DOCKER_HUB="${ISTIO_DOCKER_HUB}"
 fi
 
-# used by pilot/istioctl/security version package
-echo buildAppVersion    "${VERSION}"
+# used by pkg/version
+echo buildVersion       "${VERSION}"
```
Comment (mattdelco) [inline bin/get_workspace_status]: If you really want to change this name then I think you'll also need to make a similar name change for buildAppVersion in pkg/version/version.go

**Comment by geeknoid** [comment]
Ah shoot.

**Comment by geeknoid** [comment]
Updated...

File: bin/get_workspace_status [hunk 2]
```diff
@@ -51,14 +51,10 @@ if [[ -n ${ISTIO_DOCKER_HUB} ]]; then
...
 echo buildGitRevision   "${BUILD_GIT_REVISION}"
 echo buildUser          "$(whoami)"
 echo buildHost          "$(hostname -f)"
 echo buildDockerHub     "${DOCKER_HUB}"
-
-# used by mixer/broker version package
-echo "buildVersion ${VERSION}"
-echo "buildStatus ${tree_status}"
-echo "buildID $(TZ=UTC git log --pretty="%cd-%h" --date=short-local  -n 1)"
+echo buildStatus        ${tree_status}"
```
Comment (mattdelco) [inline bin/get_workspace_status]: Hrm, so this was 3 quotes on this line, now 1.  How 'bout 2?

**Comment by geeknoid** [comment]
Argh, my editor is playing games with me. Should be fixed now.

**Comment by ldemailly** [comment]
circle ci doesn't check the commands, you have to rerun manually there
(it also doesn't check against master, unlike prow, so if it's fixed on master you have to waste time manually rebasing and then you lose the lgtm...)

**Pull request merged**